arch/xtensa/src/common/espressif/esp_wireless.c

```diff
@@ -1520,7 +1520,7 @@ int32_t esp_wifi_init(const wifi_init_config_t *config)
   ret = coex_init();
   if (ret)
     {
-      wlerr("ERROR: Failed to initialize coex error=%d\n", ret);
+      wlerr("ERROR: Failed to initialize coex error=%ld\n", ret);
```

Comment:
it seems inconsistent to keep `ret` int32_t while changing the type of the function int.

(i guess int is better as it matches esp_err_t)